packages/eslint-plugin-pf-codemods/test/rules/nav-list-variant.js

```diff
@@ -0,0 +1,75 @@
+const ruleTester = require('./ruletester');
+const rule = require('../../lib/rules/nav-list-variant');
+
+const variantVal = '"horizontal" || "default" || "tertiary"'
+
+ruleTester.run("nav-list-variant", rule, {
+  valid: [
+    {
+      code: `import { Nav, NavList } from '@patternfly/react-core';
+<Nav variant="tertiary">
+    <NavList>
+        some item
+    </NavList>
+</Nav>`
+    }
+  ],
+  invalid: [
+    {
+      code: `import { Nav, NavList } from '@patternfly/react-core';
+<Nav>
+    <NavList variant="default">
+        some item
+    </NavList>
+</Nav>`,
+      output: `import { Nav, NavList } from '@patternfly/react-core';
+<Nav variant="default">
+    <NavList >
+        some item
+    </NavList>
+</Nav>`,
+      errors: [{
+        message: `variant has been removed from NavList, use <Nav variant=${variantVal}> instead`,
+        type: "JSXElement",
+      }]
+    },
+    {
+      code: `import { Nav, NavList } from '@patternfly/react-core';
+<Nav>
+    <NavList variant="tertiary">
+        some item
+    </NavList>
+</Nav>`,
+      output: `import { Nav, NavList } from '@patternfly/react-core';
+<Nav variant="tertiary">
+    <NavList >
+        some item
+    </NavList>
+</Nav>`,
+      errors: [{
+        message: `variant has been removed from NavList, use <Nav variant=${variantVal}> instead`,
+        type: "JSXElement",
+      }]
+    },
+    {
+      code:   `import { Nav, NavList } from '@patternfly/react-core';
+<Nav>
+    <NavList variant="horizontal">
+        some item
+    </NavList>
+</Nav>`,
+      output: `import { Nav, NavList } from '@patternfly/react-core';
+<Nav variant="horizontal">
+    <NavList >
+        some item
+    </NavList>
+</Nav>`,
+      errors: [{
+        message: `variant has been removed from NavList, use <Nav variant=${variantVal}> instead`,
+        type: "JSXElement",
+      }]
+    }
```

Comment:
Can you add this silly test case?
```jsx
import { Nav, NavList } from '@patternfly/react-core';
<Nav>
    <NavList variant>
        some item
    </NavList>
</Nav>
```

I think currently this rule would not produce valid JSX since `context.getSourceCode().getText(variantAttr)` would return an empty string.